### DISCUSSION TITLE
Model for Anki Vector

### DISCUSSION THREAD

**Discussion by HeChengHui** [cyberbotics/webots#3716] [Q&A]
I am currently trying to do some sims using the Anki Vector robot. However, I can't find the model in Webots even though the documentation mentioned [it](https://randym32.github.io/Anki.Vector.Documentation/tools/Webots.html). Does anyone knows how can I import this robot's model into Webots?

**Comment by omichel** [comment]
The Anki Cozmo model your mention is not included in the Webots distribution. Unfortunately, it is not publicly available.

**Reply by HeChengHui** [reply]
I see. Is it possible to use another model and somehow transfer it over to Anki?

**Reply by omichel** [reply]
Yes, but you will have to develop it on your own.
